lib/morpheus.gi

```diff
@@ -2859,6 +2897,10 @@ local m;
     if m<>fail then return m;fi;
   fi;
 
+  if Size(G)>50 and IsPGroup(G) and LoadPackage("anupq")=true then
```

Comment:
So far the policy for the GAP library always has been that we don't load packages, we only use them if already loaded. That would suggest to do this:
```suggestion
  if Size(G)>50 and IsPGroup(G) and IsPackageMarkedForLoading("anupq","")=truethen
```

Of course we can revisit this policy and discuss if this is still what we want, but perhaps that should be done on the GAP mailing list and not in a pull request discussion?

Reply:
Note that we do load `anupq` by default if it is compiled, so in practice there won't be much of a difference either way...

Reply:
Oh wait, that's actually wrong, I confused it with `autpgrp`. So `anupq` is *not* loaded by default. Apologies for the confusion.